### PR TITLE
[Docs] Scheduler doc EN version.

### PR DESCRIPTION
**Pull request by RangiLyu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/tutorials/param_scheduler.md [hunk 1]
```diff
@@ -1,3 +1,228 @@
...
+One of the simplest and most common learning rate adjustment strategies is multi-step learning rate decay, which reduces the learning rate to a fraction at regular intervals.
+PyTorch provides LRScheduler to implement various learning rate adjustment strategies. In MMEngine, we have extended it and implemented a more general [ParamScheduler](mmengine.optim._ParamScheduler).
+It can adjust optimization hyperparameters such as learning rate and momentum. It also supports the combination of multiple schedulers to create more complex scheduling strategies.
+
+## Usage
+
+We first introduce how to use PyTorch's `torch.optim.lr_scheduler` to adjust learning rate.
+
+<details>
+<summary>如何使用 PyTorch 内置的学习率调度器调整学习率</summary>
```
Comment (C1rN09) [inline docs/en/tutorials/param_scheduler.md:13]:
```suggestion
<summary>How to use PyTorch's builtin learning rate scheduler?</summary>
```

File: docs/en/tutorials/param_scheduler.md [hunk 2]
```diff
@@ -1,3 +1,228 @@
...
+        output = model(data)
+        loss = 1 - output
+        loss.backward()
+        optimizer.step()
+    scheduler.step()
+```
+
+</details>
+
+`mmengine.optim.scheduler` supports most of PyTorch's learning rate schedulers such as `ExponentialLR`，`LinearLR`，`StepLR`，`MultiStepLR`, etc. All of the supported schedulers please refer to [parameter scheduler API documentation](https://mmengine.readthedocs.io/en/latest/api/optim.html#scheduler).
```
Comment (C1rN09) [inline docs/en/tutorials/param_scheduler.md:40]:
```suggestion
`mmengine.optim.scheduler` supports most of PyTorch's learning rate schedulers such as `ExponentialLR`，`LinearLR`，`StepLR`，`MultiStepLR`, etc. Please refer to [parameter scheduler API documentation](https://mmengine.readthedocs.io/en/latest/api/optim.html#scheduler) for all of the supported schedulers.
```

File: docs/en/tutorials/param_scheduler.md [hunk 3]
```diff
@@ -1,3 +1,228 @@
...
+
+MMEngine also supports adjust momentum with parameter schedulers. To use momentum schedulers, replace `LR` in the class name to `Momentum`, such as `ExponentialMomentum`，`LinearMomentum`. Further, we implement the general parameter scheduler ParamScheduler, which is used to adjust the specified hyperparameters in the optimizer, such as weight_decay, etc. This feature makes it easier to practice some complex hyperparameter tuning strategies.
```
Comment (C1rN09) [inline docs/en/tutorials/param_scheduler.md:42]:
```suggestion
MMEngine also supports adjusting momentum with parameter schedulers. To use momentum schedulers, replace `LR` in the class name to `Momentum`, such as `ExponentialMomentum`，`LinearMomentum`. Further, we implement the general parameter scheduler ParamScheduler, which is used to adjust the specified hyperparameters in the optimizer, such as weight_decay, etc. This feature makes it easier to apply some complex hyperparameter tuning strategies.
```

File: docs/en/tutorials/param_scheduler.md [hunk 4]
```diff
@@ -1,3 +1,228 @@
...
+If using the runner with the registry and config file, we can specify the scheduler by setting the `param_scheduler` field in the config. The runner will automatically build a parameter scheduler based on this field:
+
+```python
+# build the scheduler with config file
+param_scheduler = dict(type='MultiStepLR', by_epoch=True, milestones=[8, 11], gamma=0.1)
+```
+
+Note that the parameter `by_epoch` is added here, which controls the frequency of learning rate adjustment. When set to True, it means adjusting by epoch. When set to False, it means adjusting by iteration.  The default value is True.
+
+In the above example, it means to adjust according to epochs. At this time, the unit of the parameters is epoch. For example, \[8, 11\] in `milestones` means that the learning rate will be multiplied bt 0.1 times at the end of the 8 and 11 epoch.
```
Comment (C1rN09) [inline docs/en/tutorials/param_scheduler.md:79]:
```suggestion
In the above example, it means to adjust according to epochs. At this time, the unit of the parameters is epoch. For example, \[8, 11\] in `milestones` means that the learning rate will be multiplied by 0.1 at the end of the 8 and 11 epoch.
```

File: docs/en/tutorials/param_scheduler.md [hunk 5]
```diff
@@ -1,3 +1,228 @@
...
+param_scheduler = dict(type='CosineAnnealingLR', by_epoch=True, T_max=12, convert_to_iter_based=True)
+```
+
+![image](https://user-images.githubusercontent.com/12907710/185115509-d4f2156a-d940-44b7-801f-3023ab76a2d3.png)
+
+### Combine multiple LRSchedulers (e.g. learning rate warm-up)
+
+In the training process of some algorithms, the learning rate is not adjusted according to a certain scheduling strategy from beginning to end. The most common example is learning rate warm-up.
+
+For example, in the first few iterations, a linear strategy is used to increase the learning rate from a small value to normal, and then adjusted by another strategy.
```
Comment (C1rN09) [inline docs/en/tutorials/param_scheduler.md:124]:
```suggestion
For example, in the first few iterations, a linear strategy is used to increase the learning rate from a small value to normal, and then another strategy is applied.
```

File: docs/en/tutorials/param_scheduler.md [hunk 6]
```diff
@@ -1,3 +1,228 @@
...
+![image](https://user-images.githubusercontent.com/12907710/185112707-356e1c20-d104-45b6-954c-55d552aab7ac.png)
+
+If using the runner with the registry and config file, we can specify the scheduler by setting the `param_scheduler` field in the config. The runner will automatically build a parameter scheduler based on this field:
+
+```python
+# build the scheduler with config file
+param_scheduler = dict(type='MultiStepLR', by_epoch=True, milestones=[8, 11], gamma=0.1)
+```
+
+Note that the parameter `by_epoch` is added here, which controls the frequency of learning rate adjustment. When set to True, it means adjusting by epoch. When set to False, it means adjusting by iteration.  The default value is True.
```
Comment (zhouzaida) [inline docs/en/tutorials/param_scheduler.md:77]:
```suggestion
Note that the parameter `by_epoch` is added here, which controls the frequency of learning rate adjustment. When set to True, it means adjusting by epoch. When set to False, it means adjusting by iteration. The default value is True.
```

**Review by zhouzaida — APPROVED** [review]
